src/cpp/session/modules/SessionCodeSearch.cpp

```diff
@@ -1787,7 +1787,11 @@ Error searchCode(const json::JsonRpcRequest& request,
       // don't index auto-generated files
       const std::string& context = item.context();
       if (boost::algorithm::ends_with(context, "RcppExports.R") ||
-          boost::algorithm::ends_with(context, "RcppExports.cpp"))
+          boost::algorithm::ends_with(context, "RcppExports.cpp") ||
+          boost::algorithm::ends_with(context, "arrowExports.R") ||
+          boost::algorithm::ends_with(context, "arrowExports.cpp") ||
+          boost::algorithm::ends_with(context, "cpp11.R") || 
+          boost::algorithm::ends_with(context, "cpp11.cpp"))
```

Comment:
I'm actually not sure this is needed, because what we do care about is that we don't see e.g. the R functions from the read-only file. So the filtering of `"do not edit by hand"` might just be enough.